### DISCUSSION TITLE
[Script request] Komodo

### DISCUSSION THREAD

**Discussion by jcastro** [community-scripts/ProxmoxVE#390] [Request script]
### Application Name

Komodo

### Website

https://komo.do/

### Description

Deploy docker containers 🚀
Deploy containers, deploy docker compose, see uptime and logs across all your servers

Powered by Rust 🦀
The core API and periphery agent are written in Rust

Automated builds 🛠️
Build auto versioned docker images from git repos, trigger builds on git push

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by Nonobis** [comment]
A script for installing/updating agent on all LXC could be great also ....

**Comment by Solear** [comment]
+1 komodo as a alternative for dockge. I love dockge, but its a paused developement at this moment.

**Comment by ither** [comment]
Yes, please. It is so much better than dockge. Here it is compared as a Portainer alternative:
https://www.virtualizationhowto.com/2024/12/portainer-alternative-komodo-for-docker-stack-management-and-deployment/

**Comment by MrCaringi** [comment]
great idea!!!!

**Comment by MickLesk** [comment]
Ready for use / testing:

```bash
bash -c "$(wget -qLO - https://github.com/community-scripts/ProxmoxVE/raw/main/ct/komodo.sh)"
```

https://community-scripts.github.io/ProxmoxVE/scripts?id=komodo

if you see an bug, dont answer here, create an issue.
